From 04fbcede889b10b53e6c273fdba09e7e054b1078 Mon Sep 17 00:00:00 2001
From: Axel Huebl <axel.huebl@plasma.ninja>
Date: Wed, 13 Aug 2025 22:02:11 -0700
Subject: [PATCH] test_charge_deposition: SP Tolerance

Lower the tolerance slightly for `test_charge_deposition`
so it passes in SP and DP.
---
 tests/python/test_charge_deposition.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/python/test_charge_deposition.py b/tests/python/test_charge_deposition.py
index ed2e32ec..be78b898 100755
--- a/tests/python/test_charge_deposition.py
+++ b/tests/python/test_charge_deposition.py
@@ -50,7 +50,7 @@ def test_charge_deposition(save_png=True):
     dV = np.prod(dr)
 
     beam_charge = dV * rs  # in C
-    assert math.isclose(beam_charge, -1.0e-9, rel_tol=1.0e-8)
+    assert math.isclose(beam_charge, -1.0e-9, rel_tol=9.8e-7)
 
     half_z = sim.n_cell[2] // 2  # order: x,y,z
 
-- 
2.43.0

